guides/getting-started/getting-started-kube.adoc

```diff
@@ -7,7 +7,7 @@
 
 ## Before you start
 
-Make sure you have Minikube installed, ideally with the Ingress addon enabled.
+Make sure you have Minikube installed, it is running with Hyperkit or Virtualbox as driver and, possilby, the Ingress addon is enabled.
```

Comment:
Is there any reason to list the drivers? Docker also works fine e.g.

Reply:
Yes, using the Docker driver the Ingress doesn't actually works.
References:
 - https://github.com/kubernetes/minikube/issues/7332
 - https://stackoverflow.com/questions/63388065/minikube-kubernetes-wont-allow-ingress-on-mac-despite-running-as-a-vm
 
Tested locally also.

Reply:
Ingress works for me with minikube 1.24 and Docker driver as instructed by the CLI:
```
💡  After the addon is enabled, please run "minikube tunnel" and your ingress resources would be available at "127.0.0.1"
```

Reply:
In this case I would say that those instructions should be updated:
https://github.com/keycloak/keycloak-web/blob/aed1c13c1d451fb6033d96c13b3b5936263d4587/guides/getting-started/getting-started-kube.adoc#access-keycloak-with-ingress-addon-enabled

Since the ingress is not going to be exposed through `minikube ip`

Reply:
I would really not list the drivers at all. We're making implications (that only VBox nad Hyperkit work) which are simply not true. Docker works (even though that it requires a little bit different workflow) but mainly there are other drivers which for what we know might work OOTB.

Reply:
> I would really not list the drivers at all

Done.